js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
```

Comment:
Okay. It seems the consensus from @jjeffryes and @rmisio is aggregate the views. And this approach with separating the views ran into some issues with nested models and so on. Seriously considering just making 1 view. It's a good point about the flattened names. Thanks @rmisio.